### DISCUSSION TITLE
Two Hot Takes for fast4x:

### DISCUSSION THREAD

**Discussion by dAtA-TRoN** [fast4x/RiMusic#1498] [General]
**Hot Take 1: Is it time to say goodbye to the sidebar navigation?**

While giving users more control over the app's layout sounds good on paper, ensuring it functions properly seems to require a lot of work. Moreover, on those small phone screens, it appears more like a fancy gimmick than something genuinely useful, as the design often interferes with usability and wastes space. Personally, I've always considered this a poor design choice, adding complexity to your job, making implementation harder, and potentially causing problems later on. Dealing with two different types of button pairs, such as text buttons versus icon buttons, only exacerbates the situation. I understand that it might be disheartening for you to toss aside so much work, but I believe that if you don't, it could lead to even more work for you in the future. Sometimes, simplicity is best, and it might be wiser to simplify and streamline rather than complicate matters just to please everyone.

---

**Hot Take 2: Advocating for a Feature Freeze**

With a multitude of concepts already in consideration, it's vital to prioritize refining and polishing current features. It may be prudent to transition from introducing fresh functionalities to improving user experience and resolving any persistent issues. From an external perspective, it appears that you're inundated with suggestions (and I'm certainly culpable of contributing to that), fostering a sense of grappling with an incessant barrage of requests. I think we've covered enough ground and touched on all the important points by now. Perhaps it's time to focus on enhancing what's already there.

**Comment by dAtA-TRoN** [comment]
@fast4x I don't know if you're seeing new discussions. Aside from this one, I've created another one. I just wanted to let you know.

**Reply by fast4x** [reply]
Hi yes I see
